src/main/java/com/google/sps/servlets/JoinSessionServlet.java

```diff
@@ -22,34 +22,40 @@
 import com.google.sps.data.AttendeeInterface;
 import com.google.sps.data.Attendee;
 
-/** Servlet that returns a list of attendees in a session.  */
-@WebServlet("/join")
+/** 
+ * Servlet that stores users data in datastore and redirected them to the 
+ * session webpage.  
```

Comment:
Vague. This could describe almost every webpage on the internet. What exactly does this servlet do?